charts/vlm/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ .Chart.Name }}-deny-egress
+  labels:
+    {{- include "vlm.labels" . | nindent 4 }}
+spec:
+  podSelector:
+    matchLabels:
+      {{- include "vlm.selectorLabels" . | nindent 4 }}
+  policyTypes:
+    - Egress
```

Comment:
correct, the VLM should not need to access anything outside its pod so blocking all egress means it can not have access to any data its not set up with during the pod init